isminet/models/base.py

```
@@ -67,17 +81,61 @@ def validate_negative(cls, v: Optional[int]) -> Optional[int]:
+    """Meta information for UniFi Network API responses."""
+
+    rc: str = "ok"
+    msg: Optional[str] = None
+
+
+class BaseResponse(BaseModel, Generic[T]):
+    """Base response model for UniFi Network API."""
+
+    model_config = ConfigDict(extra="allow")
+
+    meta: Dict[str, Any]
```

Ruff (0.8.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)